forge-game/src/main/java/forge/game/ability/effects/DecreaseSpeedEffect.java

```diff
@@ -0,0 +1,15 @@
+package forge.game.ability.effects;
+
+import forge.game.ability.SpellAbilityEffect;
+import forge.game.player.Player;
+import forge.game.spellability.SpellAbility;
+
+public class DecreaseSpeedEffect extends SpellAbilityEffect {
```

Comment:
@Hanmac
imo a merge with IncreaseSpeed into a combined effect would make sense?

Reply:
hm yeah, as One Effect it might be better?

Reply:
Sounds good to me,